libraries/stdlib/src/kotlin/time/Duration.kt

```diff
@@ -42,6 +42,119 @@ public inline class Duration internal constructor(internal val value: Double) :
         /** Converts the given time duration [value] expressed in the specified [sourceUnit] into the specified [targetUnit]. */
         public fun convert(value: Double, sourceUnit: DurationUnit, targetUnit: DurationUnit): Double =
             convertDurationUnit(value, sourceUnit, targetUnit)
+
+        /** Returns a [Duration] equal to this [Int] number of nanoseconds. */
```

Comment:
Here and everywhere below: 

`Returns a [Duration] equal to this [Int] number of ...`

Probably should be 
`Returns a [Duration] equal to [value] number of ...`
or
`Returns a [Duration] equal to [Int] [value] number of ...`


as we no longer have a receiver